### PR TITLE
fix(curriculum): Explicitly tell campers to use the > selector

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Related to #47554

It explicitly states to use the greater than selector.

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614090d5a22b6f0a5a6b464c.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ dashedName: step-13
 
 # --description--
 
-Target unordered list elements within `nav` elements, and use _Flexbox_ to evenly space the children.
+Use the `>` selector to target unordered list elements within `nav` elements, and use _Flexbox_ to evenly space the children.
```
Comment (CBID2) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614090d5a22b6f0a5a6b464c.md:13]: [Proofreading] Adding the definite article "the" would help campers gain a better understanding of the elements that need the selector. 
```suggestion
Use the `>` selector to target the unordered list elements within `nav` elements, and use _Flexbox_ to evenly space the children.

```

**Review by CBID2 — COMMENTED** [review]
Hey, @a2937.Your PR looks pretty good. I just gave a suggestion on how to make the direction a bit clearer. I hope you find them helpful! 😄  👍🏾

**Review by CBID2 — COMMENTED** [review]
LGTM

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Comment by a2937** [comment]
@CBID2 I think you still need to approve the changes. [GitHub docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews)

**Comment by CBID2** [comment]
> @CBID2 I think you still need to approve the changes. [GitHub docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews)

I don't have the rights to do that. Sorry @a2937 😞

**Review by naomi-lgbt — APPROVED** [review]

**Comment by naomi-lgbt** [comment]
Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Pull request merged**